README.md

```diff
@@ -0,0 +1,33 @@
+# Codice NSILI 
+
+## Codice NSILI Features 
+
+Federated Source and Endpoint implementations of **STANAG 4559** to allow interoperability with other systems that implement the NSILI specification.
```

Comment:
Wonder if we have have a good link to the specification, although not sure if that would create unnecessary maintenance on an external link.

Reply:
I looked for one, and only found copies for sale by third parties so I let it go